src/input/cursor.c

```diff
@@ -158,6 +161,32 @@ request_cursor_notify(struct wl_listener *listener, void *data)
 	}
 }
 
+static void
+request_set_shape_notify(struct wl_listener *listener, void *data)
+{
+	struct wlr_cursor_shape_manager_v1_request_set_shape_event *event = data;
+	struct seat *seat = wl_container_of(listener, seat, request_set_shape);
+	struct wlr_seat_client *focused_client = seat->seat->pointer_state.focused_client;
+
+	/* Prevent setting a cursor image when moving or resizing */
+	if (seat->server->input_mode != LAB_INPUT_STATE_PASSTHROUGH) {
+		return;
+	}
+
+	/*
+	 * This can be sent by any client, so we check to make sure this one is
+	 * actually has pointer focus first.
```

Comment:
Nitpick: spelling, `s/ is$//`

Also in the source of the comment.

Reply:
Fixed. It's a copypasta, so the one above has the same typo, fixed in another commit.